### PR TITLE
transfer readme correction

### PR DESCRIPTION
**Pull request by ankitdas13**
Doc reference : https://razorpay.com/docs/api/payments/route/

### CODE REVIEW

**Review by sofiya02 — CHANGES_REQUESTED** [review]

File: documents/transfer.md [hunk 1]
```diff
@@ -11,36 +11,87 @@ $api->payment->fetch($paymentId)->transfer(array('transfers' => array(array('acc
 | Name          | Type        | Description                                 |
 |---------------|-------------|---------------------------------------------|
 | paymentId*   | string      | The id of the payment to be fetched  |
-| transfers   | array     | All parameters listed [here](https://razorpay.com/docs/api/route/#create-transfers-from-payments) are supported |
+| transfers["account"]*   | string     | The id of the account to be fetched   |
```
[the remark this answers is not in the document]
Comment (ankitdas13) [reply documents/transfer.md:14]: done

[the remark this answers is not in the document]
Comment (ankitdas13) [reply documents/transfer.md:14]: done

**Review by sofiya02 — CHANGES_REQUESTED** [review]
Check response of all api

**Review by sofiya02 — CHANGES_REQUESTED** [review]
check all api with response as mentioned in excel

**Review by sofiya02 — CHANGES_REQUESTED** [review]
please check the excel and update respective response

**Review by sofiya02 — APPROVED** [review]
LGTM

**Pull request merged**